TokenService.js

```diff
@@ -67,8 +67,12 @@ export class TokenService {
     payload.authenticationMethod = authenticationMethod;
     payload.requiredAuthenticationMethods = requiredAuthenticationMethods;
 
-    const response = await axios.post(url + `/${type}`, payload);
-    return {result: response.data};
+    try {
+      const response = await axios.post(url + `/${type}`, payload);
```

Comment:
We need to replace axios with db's http-client. We may as well do that here instead of trying to make axios work (which must be removed due to vulnerabilities anyway).